config/locales/filters.en.yml

```diff
@@ -39,6 +45,9 @@ en:
       filter_date_responded: Filter by date of case response
       filter_exemption: Filter by exemptions
       filter_sensitivity: Filter by sensitivity
+      filter_complaint_type: Filter by complaint ype
```

Comment:
missing letter ;) 

```suggestion
      filter_complaint_type: Filter by complaint type
```

Reply:
@Polsonby just looked at the ticket again, yes, I think I misunderstood the sensitivity,  let me change it